pkg/state/proof.go

```diff
@@ -0,0 +1,116 @@
+package state
+
+import (
+	"bytes"
+
+	"github.com/IBM-Blockchain/bcdb-server/pkg/crypto"
+	"github.com/IBM-Blockchain/bcdb-server/pkg/types"
+	"github.com/pkg/errors"
+)
+
+var DeleteBytes = []byte{1}
```

Comment:
This means "Key Deletion Marker" or "Key Deletion Tombstone" right? 
I suggest to rename to a more descriptive name...

Reply:
Ok